src/app/threat-report-overview/add-external-report/external-report-form.ts

```diff
@@ -2,10 +2,10 @@ import { FormGroup, FormControl, Validators, FormArray } from '@angular/forms';
 
 export const ExternalReportForm = () => new FormGroup({
     external_ref_name: new FormControl('', Validators.required),
-    external_ref_source_name: new FormControl('', Validators.required),
-    external_ref_external_id: new FormControl('', Validators.required),
-    external_ref_description: new FormControl(),
-    external_ref_url: new FormControl('', Validators.required),
+    external_ref_source_name: new FormControl(''),
```

Comment:
source name is required, per stix, https://docs.google.com/document/d/1dIrh1Lp3KAjEMm8o2VzAmuV0Peu-jt9aAh1IHrjAroM/edit#